frontend/src/pages/auth/auth.tsx

```diff
@@ -59,7 +59,7 @@ const Auth: React.FC = () => {
     <PageLayout>
       <div className={styles['page-content']}>
         <div className={styles['logo-wrapper']}>
-          <span className={styles['logo-dot']} />
+          <Icon iconName="logo" />
```

Comment:
it should be an image

Reply:
You mean to use image element or I actually need to import out logo as .png file?